library/Result.php

```diff
@@ -39,8 +39,8 @@ public function __construct(
         public readonly Mode $mode = Mode::DEFAULT,
         ?string $name = null,
         ?string $id = null,
+        public readonly string|int|null $path = null,
```

Comment:
I could set the value of this property with the value of id, but then I wouldn't be able to identify those results that easily -- although I could check if the path is not the same as the id... Food for thought